chapters/data-collection.tex

```diff
@@ -1,362 +1,251 @@
 
-CAPI surveys\sidenote{\url{https://dimewiki.worldbank.org/wiki/Computer-Assisted_Personal_Interviews_(CAPI)}}
-are primarily created in Excel, Google Sheets,
-or software-specific form builders
-making them one of the few research outputs for which little coding is required.\sidenote{\url{https://dimewiki.worldbank.org/wiki/SurveyCTO_Coding_Practices}}
-The \texttt{ietestform} command,\sidenote{\url{https://dimewiki.worldbank.org/wiki/ietestform}} part of
-\texttt{iefieldkit}, implements form-checking routines
-for \textbf{SurveyCTO}, a proprietary implementation of \textbf{Open Data Kit (ODK)}.
-However, since they make extensive use of logical structure and
-relate directly to the data that will be used later,
-both the field team and the data team should
-collaborate to make sure that the survey suits all needs.\cite{krosnick2018questionnaire}
-
-Generally, this collaboration means building the experimental design
-fundamentally into the structure of the survey.
-In addition to having prepared a unique anonymous ID variable
-using the master data,
-that ID should be built into confirmation checks in the survey form.
-When ID matching and tracking across rounds is essential,
-the survey should be prepared to verify new data
-against \textbf{preloaded data} from master records or from other rounds.
-\textbf{Extensive tracking} sections --
-in which reasons for \textbf{attrition}, treatment \textbf{contamination}, and
-\textbf{loss to follow-up} can be documented --
+There is debate over how individual questions should be identified: formats like \texttt{hq\_1} are hard to remember and unpleasant to reorder, but formats like \texttt{hq\_asked\_about\_loans} quickly become cumbersome.
+We recommend using descriptive names with clear prefixes so that variables within a module stay together when sorted alphabetically.\sidenote{\url{https://medium.com/@janschenk/variable-names-in-survey-research-a18429d2d4d8}} Variable names should never include spaces or mixed cases (all lower case is best). Take care with the length: very long names will be cut off in certain software, which could result in a loss of uniqueness. We discourage explicit question numbering, as it discourages re-ordering, which is a common recommended change after the pilot. In the case of follow-up surveys, numbering can quickly become convoluted, too often resulting in variables names like 'ag\_15a', 'ag\_15\_new', ag\_15\_fup2', etc.
+
+Questionnaires must include ways to document the reasons for \textbf{attrition}, treatment \textbf{contamination}, and \textbf{loss to follow-up}.
```

Comment:
**loss to follow-up** is written **loss-to-follow-up** in previous chapters. Which one should we use?

Reply:
I prefer loss to follow-up. @bbdaniels ?

Reply:
"loss to follow-up" works